lib/linux_admin/network_interface.rb

```diff
@@ -148,7 +148,7 @@ def start
     #
     # @return [Boolean] whether the command succeeded or not
     def stop
-      Common.run(Common.cmd("ifdown"), :params => [@interface]).success?
+      Common.run(Common.cmd("ifconfig"), :params => [@interface, "down"]).success?
```

Comment:
So if i use `ip` to shutdown/start an interface, and not use `ifup & ifdown` it should be fine. Let me test.